src/main.ts

```diff
@@ -110,62 +123,102 @@ function pauseRecording() {
   }
 }
 
-
-/** Stop the current recording session. */
+/**
+ * Stop recording.
+ */
 function stopRecording() {
   isRecording = false;
   isPaused = false;
   setControls(false, true, true, recordedNotes.length === 0);
   if (pauseBtn) pauseBtn.textContent = "Pause";
 }
 
-/** Start playback of recorded notes. */
+/**
+ * Trim leading silence from the recording.
+ */
+function trimRecording() {
+  if (recordedNotes.length === 0) return;
+  const firstTime = recordedNotes[0].time;
+  recordedNotes = recordedNotes.map(note => ({
```

Comment:
Not sure if this will really trim the silence from the recording. all that this is doing is, convert the absolute time stamps to relative timestamps with respect to first note right?